modules/overview-of-pipeline-runs.adoc

```diff
@@ -4,21 +4,24 @@
 = Overview of pipeline runs
 
 [role='_abstract']
-A pipeline run is a single execution of a data science pipeline. As data scientist, you can use {productname-short} to define, manage, and track executions of a data science pipeline. You can view a record of your data science project's previously executed and scheduled runs from the *Runs* page in the {productname-short} user interface.
+A pipeline run is a single execution of a data science pipeline. As data scientist, you can use {productname-short} to define, manage, and track executions of a data science pipeline. You can view a record of previously executed, scheduled, and archived runs from the *Runs* page in the {productname-short} user interface.
 
-Runs are intended for portability. Therefore, you can clone your pipeline runs to reproduce and scale them accordingly, or delete them when you longer require them. You can configure a run to execute only once immediately after creation or on a recurring basis. Recurring runs consist of a copy of a pipeline with all of its parameter values and a run trigger. A run trigger indicates when a recurring run executes. You can define the following run triggers:
+You can optimize your use of pipeline runs for portability. You can clone your pipeline runs to reproduce and scale them accordingly, or archive them when you want to retain a record of their execution, but no longer require them. You can delete archived runs that you no longer want to retain, or you can restore them to their former state. 
```

Comment:
"to their former state (active or scheduled)"
?

Reply:
active